mscore/plugin/api/elements.h

```diff
@@ -88,6 +88,20 @@ class Element : public Ms::PluginAPI::ScoreElement {
        * \since 3.3
        */
       Q_PROPERTY(Ms::PluginAPI::Element* parent READ parent)
+      /**
+       * Computed X-axis layout position from a reference position.
```

Comment:
Could it perhaps be better to say that this is *actual* position rather than *computed* to denote that it includes also `offset` and all other possible modifiers as its part? Or, as another option, perhaps we could explicitly mention the fact that this already includes any offset applied.

Reply:
I've updated the api description.
-Dale